docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -132,13 +132,54 @@ You can configure the `expose-externally` field with the following properties:
 ** `NodePortNodeName`: Hazelcast clients are inside the same network as the Kubernetes nodes, so clients can access a member by the hostname of its Kubernetes node.
 ** `LoadBalancer`: Hazelcast clients are outside the Kubernetes network and can access members via the the external IP assigned to the LoadBalancer service.
 
+== Start Management Center
+
+You can monitor the Hazelcast cluster by starting the Management Center. You can create `ManagementCenter` Custom Resource file as `management-center.yaml`.
+
+[source,yaml]
+----
+include::ROOT:example$/operator/management-center.yaml[]
+----
+
+WARNING: Parameters `persistence` and `hazelcastClusters` should only be set at creation of the Custom Resource. Any change to `persistence` will not affect the Management Center and any change to `hazelcastClusters` will force Management Center pod to be restarted.
```

Comment:
I think the problem is not only that the MC Pod is restarted, but after the restart it may not work correctly. Or may not work at all.

Reply:
I tested it. If we add a cluster that the MC already monitors, it only gives a warning at the start. If we add a cluster that it cannot connect, nothing happens. So, I think changing `hazelcastClusters` only causes MC pod to be restarted.

Reply:
👍